docs/api/jax.rst

```diff
@@ -0,0 +1,53 @@
+..
+    Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+
+    See LICENSE for license information.
+
+Jax
+=======
+
+Types
```

Comment:
```suggestion
Enums
```

Reply:
Remove categoried as PyTorch to make the style consistent.